### PR TITLE
[scripts] Cleanup

### PR DESCRIPTION
**Pull request by summetdev**
Overview:
---
* Fixed a typo. (https://github.com/Anuken/Mindustry/commit/118bb5a5abb5dd5440349c2dbbd6265f70a675a7) (outdated)
* Replaced [Function Expressions](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/function) with arrow functions.
* Raised variables to the top of the script.
* Added semilicons.

### CODE REVIEW

File: core/assets/scripts/base.js [hunk 1]
```diff
@@ -1,36 +1,28 @@
...
+const extendContent = (classType, name, params) => new JavaAdapter(classType, params, name);
+const extend = (classType, params) => new JavaAdapter(classType, params);
+
+const readString = (path) => Vars.mods.getScripts().readString(path);
+const readBytes = (path) => Vars.mods.getScripts().readBytes(path);
+const loadMusic = (path) => Vars.mods.getScripts().loadMusic(path);
+const loadSound = (path) => Vars.mods.getScripts().loadSound(path);
+
+//these are not strictly necessary, but are kept for edge cases
+const run = (method) => new java.lang.Runnable(){run: method};
```
Comment (Anuken) [inline core/assets/scripts/base.js:18]: Why are you adding unnecessary parentheses here?

Comment (summetdev) [reply core/assets/scripts/base.js:18]: It's is recommended by many code style guidelines, for example [Airbnb](https://github.com/airbnb/javascript#arrows--one-arg-parens), also it's better for readability.
But if of course you don't like them, I can remove.

**Comment by summetdev** [comment]
@Anuken it's done.

**Comment by Anuken** [comment]
This breaks scripts (*test it out!*), and doesn't really seem necessary.

**Pull request closed without merging**